.travis.yml

```diff
@@ -1,17 +1,17 @@
 language: python
 
-dist: focal
+dist: numbat
```

Comment:
I think via the Travis docs Jammy is the latest they recommend (or at least it was the last one they documented)
https://docs.travis-ci.com/user/reference/linux/#stq=&stp=0

Reply:
Good spot 👍 I saw from here that Ubuntu 24 is out, so was curious 😄 

Will try `Jammy` after this 👍 Thanks for that 👍